lib/galaxy/tool_util/linters/outputs.py

```diff
@@ -105,10 +116,12 @@ def __check_format(node, lint_ctx, allow_ext=False):
     if fmt is None:
         fmt = node.attrib.get("format")
     if fmt == "input":
-        lint_ctx.error(
-            f"Using format='input' on {node.tag}, format_source attribute is less ambiguous and should be used instead.",
-            node=node,
-        )
+        message = f"Using format='input' on {node.tag} is deprecated. Use the format_source attribute."
+        if Version(str(profile)) <= Version("16.01"):
```

Comment:
I don't think you've imported this. Missing git add ?

Reply:
from packaging.version import Version